packages/grid/x-data-grid-pro/src/tests/columnHeaders.DataGridPro.test.tsx

```diff
@@ -266,5 +266,30 @@ describe('<DataGridPro /> - Column Headers', () => {
       expect(menu).not.toHaveFocus();
       expect(columnCell).toHaveFocus();
     });
+
+    it('should display sort column menu items as per sortingOrder prop', () => {
```

Comment:
@hanbin9775 Can you move this test to `columnHeaders.DataGrid.test.tsx` as the change is done in `x-data-grid` package?
https://github.com/mui/mui-x/blob/master/packages/grid/x-data-grid/src/tests/columnHeaders.DataGrid.test.tsx